### ISSUE TITLE
Default email from: from ADMIN_EMAIL environment

### ISSUE THREAD

**Issue by lazyest** [RocketChat/feature-requests#454]
**Is your feature request related to a problem? Please describe.**

When you creating new instance with docker it cannot send any kind of confirmation emails now. I've played with rocket.chat before and I'm sure it was better before.

**Describe the solution you'd like**

When you creating new instance with docker and have set up ADMIN_EMAIL environment variable for creating admin user while deploying I suggest to set the SMTP-From email to it as initial too. It will allow to register some new accounts in right way and maybe even not to change it after.

**Describe alternatives you've considered**
Increasing number of environment variables while deploying with docker will definitely much more difficult.

**Additional context**
I understood that most of the people didn't deploy instances every day but easy installation will 100% create more positive first impression.

**Comment by lazyest** [comment]
Or is there any working way to set it up out of web-interface after deploy? Because setting environment variable
Mail_From or MAIL_FROM didn't help.

SMTP>> MAIL FROM:<> SIZE=1728
SMTP>> RCPT TO:<root@empty>

**Comment by lazyest** [comment]
it was enough to set environment variable Mail_From while deploy. Sorry guys,it was very unclear for one who is not much familar with product to find out proper env variable name and so on. After some own research I've done all things and its working. Anyhow I'm still thinking way I've described is more elegant but I've found solution. Must say thanks to this thread https://forums.rocket.chat/t/environment-variable-setting-ids/1741
